src/widget.ts

```diff
@@ -619,7 +620,7 @@ export class DSVEditor extends Widget {
    * @param change The change args for the Datagrid (may be null)
    */
   public updateLitestore(update?: DSVEditor.ModelChangedArgs): void {
-    const selection = this._grid.selectionModel.currentSelection();
```

Comment:
I think this line might still be necessary for handling `setData` changes that don't go through the `_changeModel` method.

Reply:
Let's try and see if replicate this case and then see if we need it or not